packages/engine/src/assets/functions/LoadGLTF.ts

```diff
@@ -71,6 +74,50 @@ export async function LoadGLTF(url: string): Promise<LoadGLTFResultInterface> {
   })
 }
 
+/**
+ * Loads an Instanced Asset which is in GLTF format and uses EXT_mesh_gpu_instancing extension.
+ *
+ * @param url URL of the asset.
+ * @returns a promise of {@link LoadGLTFResultInterface}.
+ */
+export async function LoadInstancedGLTF(url: string): Promise<LoadGLTFResultInterface> {
```

Comment:
Logic in this function is similar to the logic in Asset Loader. Check if these 2 can be merged, otherwise PR is good.

Reply:
```suggestion
export async function loadInstancedGLTF(url: string): Promise<LoadGLTFResultInterface> {
```